vsphere/assets/configuration/spec.yaml

```
@@ -277,6 +277,8 @@ files:
           Examples:
           - os_type:windows
           - application_name:my_app
+          Note: Tag collection works only for vSphere 6.5 and above.
```

```suggestion
          Note: Tag collection works only for vSphere 6.0 and above.
``` 
?

https://github.com/DataDog/integrations-core/pull/8864#issuecomment-802689380